aptos-move/block-executor/src/executor.rs

```diff
@@ -347,6 +351,44 @@ where
         }
 
         ret.resize_with(num_txns, E::Output::skip_output);
-        Ok(ret)
+        Ok(ret.into_iter().map(|out| (out, vec![])).collect())
+    }
+
+    pub fn execute_block(
+        &self,
+        executor_arguments: E::Argument,
+        signature_verified_block: Vec<T>,
+        base_view: &S,
+    ) -> Result<Vec<(E::Output, Vec<(T::Key, WriteOp)>)>, E::Error> {
+        let mut ret = if self.concurrency_level > 1 {
+            self.execute_transactions_parallel(
+                executor_arguments,
+                &signature_verified_block,
+                base_view,
+            )
+        } else {
+            self.execute_transactions_sequential(
+                executor_arguments,
+                &signature_verified_block,
+                base_view,
+            )
+        };
+
+        if matches!(ret, Err(Error::ModulePathReadWrite)) {
```

Comment:
This error cannot return from execute_transactions_sequential, right?

Reply:
no it can't